### ISSUE TITLE
Small UI improvements

### ISSUE THREAD

**Issue by Destroy666x** [mybb/mybb#1583]
Front-end:
- [x] move Mark Read to footer: http://community.mybb.com/thread-154747.html
- [x] newreply.php's <i>Read the whole thread.</i> should be opened in a new tab so that changes in editor won't be lost.
- [ ] http://community.mybb.com/thread-155549.html - not sure if all are possible
- [x] add "Search this IP in SFS database" to moderation.php?action=getip
- [x] add the IP somewhere so that mods can see it when a user has 0 posts: http://community.mybb.com/thread-159790-post-1106271.html#pid1106271
- [x] add the current edit reason to quick edit's textbox, just like in editpost.php
- [ ] add link to the tracked message: http://community.mybb.com/thread-155582.html
- [ ] select code/php block
- [ ] link to all warnings in warning PM (if user can view warnings) and to buddy page in buddy request PM
- [x] add full thread titles to `title` attributes wherever it's missing (e.g. portal.php)

ACP:
- [x] server software info in ACP dashboard http://community.mybb.com/thread-166128.html
- [x] display `shown` column for registration questions
- [x] print recovery codes button
- [ ] difference numbers as a tooltip for language comparison
- [ ] display security issues http://community.mybb.com/thread-148259.html
- [ ] more to be added (?)
- [x] SFS search links for spam log
- [ ] Add expand all template groups: https://community.mybb.com/thread-209454.html

**Comment by Cu8eR** [comment]
@Destroy666x - print recovery button - it should be placed here https://github.com/mybb/mybb/issues/1843

I would be nice to see some of them in MyBB 1.8.5 :)

**Comment by Destroy666x** [comment]
Doesn't really matter where it's placed - it's a small UI improvement too so it can stay here.

All will be done in 1.8.5, they're 1-5 line changes (except http://community.mybb.com/thread-155549.html perhaps).

**Comment by Cu8eR** [comment]
@Destroy666x - could you please focus on this?

**Comment by Ben-Conway** [comment]
At this stage of 1.8 I don't think there is much point of doing these, especially when we are focusing on 2.0 even more? @euantorano @JN-Jones ?

**Comment by Cu8eeeR** [comment]
I would like to see at least some of them in 1.8 - especially for frontend

**Comment by MyBB-Andro** [comment]
Agreed with @Cu8eeeR , frontend changes would be really useful and handy. Things like: move Mark Read to footer, add the IP somewhere so that mods can see it when a user has 0 posts, select code/php block, link to all warnings in warning PM (if user can view warnings) and to buddy page in buddy request PM - are basic things to have in a forum softwares now a days. Can be achieved easily too.

Looking forward to those changes on 1.8 itself. Hope someone provides an helping hand on doing it.

**Comment by JordanMussi** [comment]
Point 6 is achieved in the fix for #2777 (#2780).

**Comment by euantorano** [comment]
Updated the checklist, thanks @JordanMussi.

**Comment by Cu8eeeR** [comment]
@euantorano - what about this one? Seems to be easy to implement and useful :)
https://community.mybb.com/thread-209454.html

**Comment by euantorano** [comment]
Added to the list @Cu8eeeR

**Comment by Ben-MyBB** [comment]
Not sure what we want to do with this issue at this stage @euantorano

**Comment by euantorano** [comment]
Yeah, I'm hoping we can vastly improve the UI and UX in 1.9 - @justinsoltesz and @Eric-Jackson are the folks to talk to about that for sure. Any backend code that's needed to improve UI and UX on the front end should be possible for 1.9.
